aioamqp/channel.py

```diff
@@ -18,6 +18,10 @@
 logger = logging.getLogger(__name__)
 
 
+def chunker(seq, size):
+    return [seq[i:size+i] for i in range(0, len(seq), size)]
```

Comment:
Why not use a generator expression instead of a list comprehension?

Reply:
which is the current implem isn't it ?

Reply:
and yes, it was a generator previously :/

Reply:
Because list comprehension is faster than iterating thru whole generator expression.
```python
def chunker(seq, size):
    return [seq[i:size+i] for i in range(0, len(seq), size)]

test_data = "a" * 100000
frame_max = 100

def test_chunker(payload, frame_max):
    for chunk in chunker(payload, frame_max):
        pass
    
def test_orig(payload, frame_max):
    for chunk in (payload[i:frame_max+i] for i in range(0, len(payload), frame_max)):
        pass
    
def test_orig_nogen(payload, frame_max):
    for chunk in [payload[i:frame_max+i] for i in range(0, len(payload), frame_max)]:
        pass

print("test_chunker")
%timeit test_chunker(test_data, frame_max)
print("test_orig")
%timeit test_orig(test_data, frame_max)
print("test_orig_list_comprehension")
%timeit test_orig_nogen(test_data, frame_max)
```
```
test_chunker
10000 loops, best of 3: 171 µs per loop
test_orig
10000 loops, best of 3: 185 µs per loop
test_orig_list_comprehension
10000 loops, best of 3: 165 µs per loop
```
Last variant is fastest because there no function calling overhead, but less readable. If developers say that it is better to use inline variant I will rewrite it.